x/epoching/keeper/keeper.go

```diff
@@ -0,0 +1,186 @@
+package keeper
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	db "github.com/tendermint/tm-db"
+)
+
+var (
+	NextEpochActionID      = []byte("next_epoch_action_id")
```

Comment:
Is it worth to have this long prefix? A byte could do the same.

Reply:
Yeah this should just be a byte